src/oom/test-oomd-util.c

```diff
@@ -41,11 +41,15 @@ static void test_oomd_cgroup_kill(void) {
         int pid[2];
         int r;
 
-        if (geteuid() != 0)
-                return (void) log_tests_skipped("not root");
+        if (geteuid() != 0) {
+                (void) log_tests_skipped("not root");
```

Comment:
It is not necessary to voidify the result.

Reply:
Hmm, do we really want this? We're replacing a two-line pattern with a four-line pattern. I don't think this is more readable or anything. And that line from coding style guidelines doesn't really apply: here we need the (void) not because the function doesn't return an error condition, but because the compiler it does and the compiler will warn otherwise.